charset_normalizer/__init__.py

```diff
@@ -45,3 +47,26 @@
     "__version__",
     "VERSION",
 )
+
+
+def set_logging_handler(
+    name: str = "charset_normalizer",
+    level: int = logging.INFO,
+    format_string: str = None,
```

Comment:
Why not setting the "%(asctime)s | %(levelname)s | %(message)s" immediatly as default? 

I read `format_string: str = None,` and not `format_string: Optional[str] = None,` Please revise.